universal-login-sdk/lib/core/utils/isBiggerThan.ts

```diff
@@ -0,0 +1,3 @@
+import {utils} from 'ethers';
+
+export const isBiggerThan = (value: utils.BigNumber, validationValue: utils.BigNumber) => value.gt(validationValue) ? value : validationValue;
```

Comment:
'Is' suggest that return type will be boolean. Maybe it should be getBigger?

Reply:
👍

Reply:
👍